docker-compose.yml

```diff
@@ -61,7 +60,6 @@ services:
     volumes:
       - .:/app
       - bundle-2.1:/usr/local/bundle
-      - gemfiles-2.1:/app/gemfiles
```

Comment:
Could you add some more details on this change? I'm not entirely sure why this needs to be removed.

Reply:
This needs to be removed because `- gemfiles-2.1:/app/gemfiles` gives each container its own `gemfiles`, because before we didn't check it into the repo, and each container generated its own set of gemfiles.

Now that it's checked in, we want `gemfiles` to be shared across all containers, which is done indirectly by `- .:/app` a few lines above.